cluster/kube-down.sh

```diff
@@ -37,4 +37,4 @@ verify-kube-binaries
 echo "... calling kube-down" >&2
 kube-down
 
-echo "Done"
+echo "Done tearing down kube"
```

Comment:
The original statement is fine and consistent with the rest of our Step/Done logs.